UnityGLTF/Assets/UnityGLTF/Scripts/Editor/GLTFImporter.cs

```diff
@@ -336,14 +336,16 @@ private GameObject CreateGLTFScene(string projectFilePath)
             ILoader fileLoader = new FileLoader(Path.GetDirectoryName(projectFilePath));
             using (var stream = File.OpenRead(projectFilePath))
             {
-                GLTFRoot gLTFRoot = GLTFParser.ParseJson(stream);
+	            GLTFRoot gLTFRoot = null;
+	            GLTFParser.ParseJson(stream, ref gLTFRoot);
                 var loader = new GLTFSceneImporter(gLTFRoot, fileLoader, stream);
 
                 loader.MaximumLod = _maximumLod;
+	            loader.isMultithreaded = true;
 
-                // HACK: Force the coroutine to run synchronously in the editor
-                var stack = new Stack<IEnumerator>();
-                stack.Push(loader.LoadScene(isMultithreaded: true));
+				// HACK: Force the coroutine to run synchronously in the editor
+				var stack = new Stack<IEnumerator>();
```

Comment:
formatting seems to be off here. Make sure you're using tabs

Reply:
Actually I'm using tabs here, but the rest of this file seems to be using spaces. Should I change the tabs to spaces?

Reply:
Probably, I think the majority of the project is using tabs.

Reply:
I think in this case whichever keeps the formatting changes to a minimum is best to see the diff.

Reply:
Issue fixed!